arches/app/datatypes/concept_types.py

```diff
@@ -209,6 +209,7 @@ def transform_value_for_tile(self, value, **kwargs):
             if value == "":
                 value = None
             else:
+                value = value.strip()
```

Comment:
I'm a little concerned that there could be existing systems with trailing spaces in their labels and that this change would break their import. 

I think the way that we could address this is rather than stripping here, we could strip in `lookup_label` doing something like:

```if concept[1] in (label, label.strip())```

That also has the advantage of striping in only once rather than in both `transform_value_for_tile` methods.

Reply:
excellent suggestion, fix in latest